code-challenges.js

```diff
@@ -46,4 +92,18 @@ const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
 const cohortList = ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
 // Expected output: ["Delta", "Echo", "Foxtrot"]
 
-// Explain your code:
+//input: array
+//output: array that returns only the 3rd, 4th, and 5th index valus
+//create a function that takes in the arrays
+// use the .slice method that will grab the index of the elements needed into a new array
+// return the values in a new array 
+
+const newArr = (array) => {
+    //let arr1 = [] // for some reason thought i needed this empty array but it wasnt after all
+    let finalArr = array.slice(3, 6) //at first i did (2, 5) but that gave me -4, 42, 5
+    return finalArr
+}
+    console.log(newArr(stockExchange)) // output [ 42, 5, -5 ]
+    console.log(newArr(cohortList)) // output [ 'Delta', 'Echo', 'Foxtrot' ]
+
+// Explain your code: the code above takes in the array. and by using .slice i was able to set that variable finalArr to only call on indexs 3-5. since i want it to stop at 5. youll see i had to put 3,6 as the arguments. once you return final Arr and console.log it gives the answer needed. i feel like this one it was the easiest way in my mind but i would love to try to use a HOF with it to map or fileter through. i just remembered the .slice from day 3 of javascript and it stuck in my mind as an easy method to use for this. 
```

Comment:
great approach - remember, get it work, then refactor.  You can definitely do this will filter! I dare you. 😆